core/src/main/resources/jenkins/monitor/JavaLevelAdminMonitor/message.jelly

```diff
@@ -0,0 +1,13 @@
+<?jelly escape-by-default='true'?>
+<j:jelly xmlns:j="jelly:core" xmlns:l="/lib/layout" xmlns:f="/lib/form">
+    <div class="alert alert-info">
+        <h5>${%Jenkins Java version}</h5>
```

Comment:
We don't have any admin monitors with headers in core. Does it make sense to add this to the presentation zoo?